modules/packages/LinearAlgebra.chpl

```diff
@@ -937,6 +937,132 @@ proc trace(A: [?D] ?eltType) {
   return trace;
 }
 
+/* Type to choose which triangle of a matrix is desired */
+enum lowerOrUpper {
```

Comment:
It'd be nice if we could get away from the `enum` arguments for the `LinearAlgebra` interface. What do you think of a `lower=false` argument instead, similar to [scipy](https://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.linalg.cholesky.html)

Reply:
I removed this one in favor of a `lower=true` argument.